src/test/java/com/epam/tc/hw3/pages/DifferentElementsPage.java

```diff
@@ -0,0 +1,56 @@
+package com.epam.tc.hw3.pages;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class DifferentElementsPage extends AbstractPage {
+
+    @FindBy(xpath = "//*[text()[contains(.,'Water')]]/input")
+    private WebElement waterCheckbox;
+    @FindBy(xpath = "//*[text()[contains(.,'Wind')]]/input")
+    private WebElement windCheckbox;
+    @FindBy(xpath = "//*[text()[contains(.,'Selen')]]/input")
+    private WebElement selenRadioButton;
+    @FindBy(className = "colors")
+    private WebElement colorsDropDownMenu;
+    @FindBy(xpath = "//option[text()='Yellow']")
+    private WebElement yellowOptionColors;
+    @FindBy(xpath = "//li[text()[contains(.,'Water: condition changed to true')]]")
```

Comment:
text() here is redundant, dot - is equal to text, this should be enough //li[contains(.,'Water: condition changed to true')]
Also, it would be good to use template and substitute the value to the one you need to choose/check
//li[contains(.,'%s: condition changed to true')]